style.css

```diff
@@ -0,0 +1,332 @@
+@import url("https://fonts.googleapis.com/css2?family=Cormorant+Infant:wght@400;600;700&family=Inter:wght@300&family=Open+Sans:wght@600&display=swap");
+
+.material-symbols-outlined {
+  font-variation-settings: "FILL" 0, "wght" 400, "GRAD" 0, "opsz" 48;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Cormorant Infant", serif;
+}
+
+body {
+  background-color: #262324;
+}
+
+a {
+  color: #fff;
+  padding: 10px;
+}
+
+a:hover {
+  color: #c1895b;
+}
+
+.menu {
+  text-decoration: none;
+  display: flex;
+  gap: 1.5rem;
+  align-content: space-between;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: nowrap;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  box-sizing: border-box;
+  border-style: solid;
+  border-color: #262324 #262324 #c1895b #262324;
+  border-bottom-width: 3px;
+  height: 7rem;
+  background-color: #262324;
+  padding: 1rem 1rem;
+  z-index: 1;
+  margin-top: 0;
+}
+
+.menu .material-symbols-outlined {
+  margin-bottom: 10px;
+}
+
+.menu-item {
+  text-decoration: none;
+  padding: 2em;
+  padding: 0.75em 2em;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.menu-item.logo {
+  padding-top: 10px;
+  display: flex;
+  justify-content: flex-start;
+  flex-grow: 0.5;
+}
+
+.footer {
+  text-align: center;
+  color: #c1895b;
+  margin-bottom: 10px;
+}
+
+/*----------Index----------*/
```

Comment:
În loc de secțiuni separate pentru fiecare pagină, ai putea face fișiere separate.

Reply:
Si CSS pentru meniu ar trebui sa se gaseasca in toate fisierele atunci?

Reply:
Fiindcă meniul apare pe toate fișierele, poți să îl păstrezi în style.css.

Adică în fiecare fișier HTML, ai încărca două de CSS: `style.css` și cel dedicat paginii (e.g. `contact.css`)